src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.4.1.0.cs

```diff
@@ -372,7 +372,6 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
     }
 
     [WcfFact]
-    [Issue(1886, OS = OSID.AnyOSX)]
```

Comment:
The 1886 issue will only track the client certificate test failures caused by  CI using root account to run the tests.

I will update the peer trust failures to the root cause issue, and remove the setup validation tests per our discussion yesterday.